docs/mkdocs/integration/hub-spoke.md

```diff
@@ -0,0 +1,363 @@
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: kmm-operator
+    namespace: openshift-operators
+spec:
+    channel: alpha
+    installPlanApproval: Automatic
+    name: kernel-module-management-hub
+    source: kmm-hub-catalog
+    sourceNamespace: openshift-marketplace
+EOF
+```
+
+4. Deploy the ACM Policy that adds the required permissions to the Spoke klusterlet, in order for the latter to be able to CRUD KMM Module CRs:
+
+```bash
+cat <<EOF | kubectl apply -f -
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+    name: allow-klusterlet-deploy-kmm-modules
+spec:
+    remediationAction: enforce
+    disabled: false
+    policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+            name: klusterlet-deploy-modules
+        spec:
+            severity: high
+            object-templates:
+            - complianceType: mustonlyhave
+            objectDefinition:
+                apiVersion: rbac.authorization.k8s.io/v1
+                kind: ClusterRole
+                metadata:
+                name: kmm-module-manager
+                rules:
+                - apiGroups: [kmm.sigs.x-k8s.io]
+                    resources: [modules]
+                    verbs: [create, delete, get, list, patch, update, watch]
+            - complianceType: mustonlyhave
+            objectDefinition:
+                apiVersion: rbac.authorization.k8s.io/v1
+                kind: RoleBinding
+                metadata:
+                name: klusterlet-kmm
+                namespace: open-cluster-management-agent
+                subjects:
+                - kind: ServiceAccount
+                name: klusterlet-work-sa
+                namespace: open-cluster-management-agent
+                roleRef:
+                kind: ClusterRole
+                name: kmm-module-manager
+                apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+    name: all-clusters-except-local
+spec:
+    clusterSelector:
+    matchExpressions:
+        - key: name
+        operator: NotIn
+        values:
+            - local-cluster
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: PlacementBinding
+metadata:
+    name: bind-klusterlet-kmm-all-clusters
+placementRef:
+    apiGroup: apps.open-cluster-management.io
+    kind: PlacementRule
+    name: all-clusters-except-local
+subjects:
+    - apiGroup: policy.open-cluster-management.io
+    kind: Policy
+    name: allow-klusterlet-deploy-kmm-modules
+EOF
+```
+
+5. Create a namespace on the Hub, in which the Build and Sign jobs will be created:
+
+```bash
+kubectl create ns kmm-jobs-test
+```
+
+## Setup the Spoke
+
+1. Build and push the KMM container image, the KMM OLM bundle and catalog from midstream repository:
```

Comment:
Same question. We have official builds - why not just use them? https://operatorhub.io/operator/kernel-module-management.

I also see now that we don't have any public subscription for kmm-hub. Should we?

Reply:
This was written before the docs were published and before 1.0 release, probably doesn't make sense anymore